src/tabs/calculations.jsx

```diff
@@ -15,6 +15,8 @@ function Calculations() {
     const { showSearch } = useSearch();
     const [targetType, setTargetType] = useState(0);
     const [refresh, setRefresh] = useState(false);
+    const [blocked, setBlocked] = useState(true);
+    const [missed, setMissed] = useState(true);
```

Comment:
It'd be better if these were settings inside `Context.settings` https://github.com/Frostiae/Flyffulator/blob/229d476c0b5fbc67e67bcaae1c56e70ddbc88018/src/flyff/flyffcontext.js#L42-L46